client/src/utils/Header/Header.tsx

```diff
@@ -0,0 +1,12 @@
+import { Link } from "react-router-dom";
```

Comment:
Headers can be a separate component. Not in Utils folder.

Reply:
yeah right I added it in the components folder